controllers/openstackcluster_controller.go

```diff
@@ -690,10 +690,10 @@ func reconcileNetworkComponents(scope *scope.WithLogger, cluster *clusterv1.Clus
 }
 
 // reconcilePreExistingNetworkComponents reconciles the cluster network status when the cluster is
-// using pre-existing networks and subnets which are not provisioned by the
+// using pre-existing networks, subnets and router which are not provisioned by the
```

Comment:
other words are plural, so either remove "s" on others or add "s" to router. thanks

Reply:
I didnt add an s to router because its only one router. Network and subnet can be multiple due to the lb and the main network and subnet.

Reply:
fair enough !